plugins/commands/create.js

```diff
@@ -22,58 +84,89 @@ module.exports = () => ({
                 security: [{ token: [] }]
             }
         },
-        handler: (request, reply) => validator(request.payload.yaml)
-            .then((config) => {
-                if (config.errors.length > 0) {
-                    throw boom.badRequest(
-                        `Command has invalid format: ${config.errors.length} error(s).`,
-                        config.errors);
+        payload: {
+            parse: true,
+            allow: ['multipart/form-data', 'application/json']
+        },
+        handler: (request, reply) => {
+            const data = request.payload;
+            let commandSpec;
+            let commandBin;
+
+            // if Content-type is multipart/form-data, both command binary and meta are posted
+            if (request.headers['content-type'].startsWith('multipart/form-data')) {
+                if (data.file === undefined || data.file.length !== 2) {
+                    return reply(boom.badRequest('Posted invalid number of files.'));
                 }
+                data.file.forEach((file) => {
+                    if (typeof file === 'string') {
+                        commandSpec = file;
```

Comment:
It also might be needed to parse `yaml` key here like you've done on non-multipart.
`commandSpec = data.yaml;`

Reply:
This is not need to fix. I'll try the implementation of that sd-cmd sends multipart with no `yaml` key, but sends application/json sends with a `yaml` key.